bin/all.sh

```diff
@@ -68,7 +68,9 @@ echo "::group::build.sh"
 bash bin/build.sh
 echo "::endgroup::"
 
+echo "::group::script.js"
```

Comment:
We might wind up with more than one JS script - how about a more descriptive name, like `artifact-upload`?